front/src/Connexion/ConnectionManager.ts

```diff
@@ -51,7 +56,7 @@ class ConnectionManager {
                 )
             );
             urlManager.pushRoomIdToUrl(room);
-            return Promise.resolve(room);
+            Promise.resolve(room);
```

Comment:
this line does nothing

Reply:
The function `initGameConnexion` is `initGameConnexion(): Promise<Room>`